File: C#/beecrowd-1008.cs
using System; 

class URI {
    static void Main(string[] args) { 
        int number;
        int workedHours;
        double salaryPerHour;
        double totalPayment;

        number = int.Parse(Console.ReadLine());
        workedHours = int.Parse(Console.ReadLine());
        salaryPerHour = double.Parse(Console.ReadLine());
        
        totalPayment = workedHours * salaryPerHour;

        Console.WriteLine(String.Format("NUMBER = {0}\nSALARY = U$ {1:0.00}", number, totalPayment));
    }
}

File: Elixir/beecrowd-1008.exs
# Input
number = IO.gets("") |> String.trim |> String.to_integer
workedHours = IO.gets("") |> String.trim |> String.to_integer
salaryPerHour = IO.gets("") |> String.trim |> String.to_float

# Processing
salary = workedHours * salaryPerHour |> :erlang.float_to_binary([decimals: 2])

# Output
IO.puts("NUMBER = #{number}")
IO.puts("SALARY = U$ #{salary}")
